alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/AlertingRestTestCase.kt

```diff
@@ -111,6 +112,19 @@ abstract class AlertingRestTestCase : ESRestTestCase() {
                 customWebhook = null)
     }
 
+    @Suppress("UNCHECKED_CAST")
```

Comment:
I think you can avoid suppressing this by using the `.stringMap()`(i believe that's what it is called) function from the test helpers.

Reply:
Change it.